UI/PauseScreen.cpp

```diff
@@ -321,6 +321,9 @@ void GamePauseScreen::CreateViews() {
 	continueChoice->OnClick.Handle<UIScreen>(this, &UIScreen::OnBack);
 
 	std::string gameId = g_paramSFO.GetValueString("DISC_ID");
+	if (gameId.empty() || gameId == "UCJS10041") {
```

Comment:
UCJS10041 as mentioned at the top of this pr is a real game - demo version of japanese Loco Roco, I think creating fakeID for one demo which doesn't need such treatment to fix a lot of homebrew is worth doing. I also never saw any homebrew with different ID, they either have none at all, or use that one, probably due to use of same tools/sdk's or whatever add's that ID.

 As for do we want it, thinking about it, I think it might not be needed everywhere, maybe even it's only needed in places that are called before running the game. I'll check and remove it from places I find.